app/src/main/res/values/strings.xml

```diff
@@ -145,6 +147,8 @@
     <string name="gy">Gy</string>
     <string name="gz">Gz</string>
     <string name="temp">Temp</string>
+    <string name="luminosity">Luminosity</string>
```

Comment:
```luminosity``` key is repeated resulting in build failing. Please resolve it and ensure build is successful.